api/proto/config/devmodel.proto

```diff
@@ -13,44 +13,31 @@ option java_package = "org.lfedge.eve.config";
 //     Constitutes set of parameters that define the physical devices.
 //     Mainly they describes all the I/O adapters that make up the device.
 //
+//  L2 objects:
+//     Objects hierarchically placed between PhysicalIO and SystemAdapters
+//     related to Layer 2 networking.
+//     Currently supported are VLANs and LAGs (Link Aggregation Groups, aka "bonds").
+//     References based on logical names are used to describe parent-child
+//     or master-slave relations between L2 objects and PhysicalIO.
 //
 //  SystemAdapters:
-//     Are the higher level contstucts these are IP-ready network end point
-//     description.  SystemAdapters used by EVE device to connect
-//     to controller, alternatively systemAdapters are used to share the
-//     network connections for applications.
+//     Higher-level IP-ready network endpoints.
+//     SystemAdapters are used by EVE device to connect to the controller
+//     and/or to share external network connectivity between applications.
 //
-//     SystemAdapters, have proxy configuration, VLAN, Wifi credentials etc.
-//     System adapter refers to PhysicalIO by name.
+//     SystemAdapters have cost, proxy configuration, Wifi credentials etc.
+//     System adapter refers to an underlying L2 object or directly to PhysicalIO
+//     by the logical name.
 //
 //   NetworkAdapters:
```

Comment:
FWIW I grepped for NetworkAdapters in the API and this seems to be a term we don't use elsewhere.
Conceptually we are presumably referring to the subset of physicalIO adapters which are of the network types.
I wonder if we should clarify this here since you clarified other existing coments.

Reply:
I think that the term `NetworkAdapters` is out-of-date. What it is trying to refer to is now `SystemAdapter`. And if this is about direct NIC assignment then it is out-of-place (should be in `devcommon.proto`). Anyway, I decided to remove that paragraph to avoid confusion.